src/index.html

```diff
@@ -2,16 +2,165 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input type="text" name="surname">
+        </label>
+
+        <label class="form_field">
+          Name:
+          <input type="text" name="name">
+        </label>
+
+        <label class="form_field">
+          How old are You?
+          <input 
+            type="number" 
+            name="old" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+
+        <label class="form_field">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="date"
+          >
+        </label>
+        <div class="form_field">
+          <label for="terms-and-conditions" >
+            I accept the term of the agreement
+          </label>
+          <input 
+            type="checkbox" 
+            name="trems" 
+            id="terms-and-conditions" 
+            class="field"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+          <input 
+            type="email" 
+            placeholder="email@example.com" 
+            name="email" 
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="6" 
+            maxlength="20" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form_field">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="cats" 
+              value="y"
```

Comment:
```suggestion
              value="yes"
```
better to specify more meaningful values